BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleAppSessionManager.m

```diff
@@ -60,6 +61,23 @@ + (NSString *)rootCacheDirGivenSharedContainerId:(NSString *)sharedContainerId
     return containerURL.path;
 }
 
++ (NSString *)tempCacheDir
+{
+    NSString *cachesPath = [NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES) lastObject];
```

Comment:
the BOXRequest requires a path, and this method is analogous to NSSearchPathForDirectoriesInDomains